src/tests/api/postEventStream.test.ts

```diff
@@ -148,4 +150,64 @@ describe("post event processor", () => {
 			},
 		]);
 	}, 20000); // timeout set to 20s to avoid infinite loop
+
+	it("Dynamo record deleted when IPV_F2F_USER_CANCEL_END event is sent", async () => {
```

Comment:
Can we make this a bit more relevant to what happens, e.g. "PII is removed and record is marked for deletion"
We should also check this flag is present on the session table after the update `accountDeletedOn`